src/views/forms/FormAggregateTransaction/FormAggregateTransactionTs.ts

```diff
@@ -192,20 +192,22 @@ export class FormAggregateTransactionTs extends FormTransactionBase {
     private createTransferTx(tx: {}): TransferTransaction {
         const maxFee = UInt64.fromUint(this.formItems.maxFee);
         this.formItems.recipientRaw = tx['formItems']['recipientRaw'];
-        const signer = PublicAccount.createFromPublicKey(tx['formItems']['signerPublicKey'], this.networkType);
+        const signer = tx['formItems']['signerPublicKey']
+            ? PublicAccount.createFromPublicKey(tx['formItems']['signerPublicKey'], this.networkType)
+            : undefined;
         const t: TransferTransaction = TransferTransaction.create(
```

Comment:
what does TransferTransaction.create do when signer is undefined?

Reply:
TransferTransaction.create() doesn't need a signer for the transaction it's an optional parameter, was just using it in case of transactions made via multi-sig accounts.